src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
Minor nitpick, but import statements here should be reordered as per the [Java coding standard](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements). I noticed similar issues in other places too.